README.md

```diff
@@ -80,7 +80,9 @@ To run integration tests you need to:
     python3 -m unittest tests.neo4j.datatypes.TestDataTypes.testShouldEchoBack
     ```
   
-  * Alternatively, It's possible to use the option `--tests RUN_SELECTED_TESTS` to build the driver backend and run the tested in the `TEST_SELECTOR` environment variable. It's specialy useful during the development of new tests when it's used in combination with `run_all.py` enabling to run one specific test against all known drivers.
+  * Alternatively, it's possible to use the option `--tests RUN_SELECTED_TESTS` to build the driver backend and run the tested in the `TEST_SELECTOR` environment variable. This will start all dependencies needed (such as neo4j or tls servers). It's especially useful during the development of new tests when used in combination with `run_all.py` enabling to run one specific test against all known drivers.
+    
+    * The command-line param `--run-only-selected <test_selector>` works as shortcut of setting the `TEST_SELECTOR` environment variable and running the command with `--tests RUN_SELECTED_TESTS`.
```

Comment:
This is very nit-picky and pleas feel free to ignore this part: I'd remove the blank line above that as the point belongs to the list above.

Another note: "~~works as~~ *is* a shortcut ~~of~~ *for*"